src/fabric/src/fabric_util.erl

```diff
@@ -152,6 +152,13 @@ all_docs_timeout() ->
 attachments_timeout() ->
     timeout("attachments", "600000").
 
+global_view_timeout(Args) ->
```

Comment:
Still seems odd to be calling this global_view_timeout when it applies to both global and partitioned.

Reply:
I've changed the name